inscriptions/ISic030018.xml

```diff
@@ -0,0 +1,226 @@
+                <change when="2020-10-08" who="#SS">Simona Stoyanova updated list of languages</change>
+                <change when="2021-04-14" who="#VM">Valentina Mignosa added metadata, text</change>
+            </listChange>
+        </revisionDesc>
+    </teiHeader>
+    <facsimile>
+        <surface/> 
+        <!--delete empty surface above and uncomment template below if needed-->
+        <!--
+  <surface type="front">
+    <graphic n="screen" url="{concat('ISic', $num, '_tiled.tif')}"
+      height="4912px" width="7360px">
+      <desc>Add Description Here</desc>
+    </graphic>
+    <graphic n="print" url="{concat('ISic', $num, '.jpg')}" height="3680px" width="5520px">
+      <desc>Add Description Here</desc>
+    </graphic>
+  </surface>
+
+    -->
+    </facsimile>
+    <text>
+        <body>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
+                <ab>
+                    <lb n="1"/><choice><corr>δᾱμοσία</corr><sic>δᾱμοσρα</sic></choice> <supplied reason="subaudible">μνᾶ</supplied>
+                </ab>
+            </div>
+            <div type="apparatus">
+                <listApp>
+                    <app>
+                        <note>Text after Arena (IGASM 2)</note>
+                    </app>
+                </listApp>
+            </div>
+            <div type="translation" xml:lang="en" resp="#VM">
+                <p></p>
+            </div>
+            <div type="translation" xml:lang="it" resp="#VM">
+                <p></p>
+            </div>
+            <div type="commentary" xml:lang="en" resp="#VM">
+                <p>Dubois points out that the same inscription is found, in Sicily, on a loom weight now in the 
+                    Museo Paolo Orsi di Siracusa (Orsi 1906, 679, no. 1, fig. 510). The weight of 418 g
+                    corresponds to two talents of silver, each weighing 208.8 g (Dubois IGDS no. 152).</p>
+            </div>
+            <div type="bibliography">
+                <listBibl type="edition">
+                    <bibl type="corpus" n="IGASM2[2]">
+                        <author>Arena</author>
+                        <date>2002</date>
+                        <citedRange>26, pl. XI.1</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/65P7AM6N"/>
+                    </bibl>
+                    <bibl type="corpus" n="IGDS1">
+                        <author>Dubois</author>
+                        <date>1989</date>
+                        <citedRange>152</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/RWXNUFEX"/>
+                    </bibl>
+                    <bibl type="bulletin" n="SEG">
+                        <citedRange>16.0547</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/F2GG87EQ"/>
+                    </bibl>
+                </listBibl>
+                <listBibl type="discussion">
```

Comment:
NB does the Orsi disscussion reference this text? If not, then I don't think we include it here